### ISSUE TITLE
Filter duplicate Tickets

### ISSUE THREAD

**Issue by mgrolinger** [tomasbjerre/git-changelog-lib#86]
Hi,

is there a possibility to filter duplicate Jira tickets? The changelog contains four "Generate changelog automatically during release process" entries, which have the same headline but different commit details. However, I'd like to shorten this to one entry  " [JIRA-1916]  Generate changelog automatically during release process"
Is there an option to do so.

```
# [unreleased] 
 [JIRA-1916]  Generate changelog automatically during release process

## [1.0.15] (31.07.2020)
 [JIRA-1916]  Generate changelog automatically during release process Generate changelog automatically during release process Generate changelog automatically during release process Generate changelog automatically during release process
 [JIRA-2119]  Service: Update Spring Boot 2.3.1 -> 2.3.2
```

**This is the template I am using:**
```
<![CDATA[
# Changelog für {{repoName}}

{{#tags}}
## [{{name}}] {{#hasTagTime}}({{tagTime}}){{/hasTagTime}}
 {{#issues}}
  {{#hasIssue}}
   {{#hasLink}}
 [{{issue}}] ({{link}}) {{title}} {{#hasIssueType}} *{{issueType}}* {{/hasIssueType}} {{#hasLabels}} {{#labels}} *{{.}}* {{/labels}} {{/hasLabels}}
   {{/hasLink}}
   {{^hasLink}}
 [{{issue}}] {{#commits}} {{{messageTitle}}}
 {{/commits}}
 {{/hasLink}}
  {{/hasIssue}}
  {{^hasIssue}}
{{#commits}}
 {{{messageTitle}}}
{{/commits}}
  {{/hasIssue}}
 {{/issues}}

{{/tags}}
]]>
```

Version of the Maven-plugin 1.61
Sincerely, Michael

**Comment by tomasbjerre** [comment]
No you cannot shorten it to one entry.

Perhaps you just want to ignore those commits? There is a feature to ignore commits based on their message:
https://github.com/tomasbjerre/git-changelog-maven-plugin/blob/master/git-changelog-maven-plugin-example/pom.xml#L250

**Comment by mgrolinger** [comment]
Hi,

thank you for your reply. Unfortunately, ignoring those commits is not an option. I think we will just keep them that way and need to consider a git squash for the next time.
Sincerely,
Michael
